swarmer/src/main/kotlin/com/gojuno/swarmer/Args.kt

```diff
@@ -27,61 +28,81 @@ private class JCommanderArgs {
+            order = 2
     )
     lateinit var pakage: String
 
     @Parameter(
             names = arrayOf("--android-abi"),
             required = true,
-            description = "Android system image abi, i.e. `google_apis/x86_64` to pass to `avdmanager create avd --abi`."
+            description = "Android system image abi, i.e. `google_apis/x86_64` to pass to `avdmanager create avd --abi`.",
+            order = 3
     )
     lateinit var androidAbi: String
 
     @Parameter(
             names = arrayOf("--path-to-config-ini"),
             required = true,
-            description = "Path either relative or absolute to the file that will be used as `config.ini` for created emulator."
+            description = "Path either relative or absolute to the file that will be used as `config.ini` for created emulator.",
+            order = 4
     )
     lateinit var pathToConfigIni: String
 
     @Parameter(
             names = arrayOf("--emulator-start-options"),
             required = false,
             variableArity = true,
-            description = "Options to pass to `emulator -avd \$emulatorName` command, i.e. `--no-window -prop persist.sys.language=en -prop persist.sys.country=US`."
+            description = "Options to pass to `emulator -avd \$emulatorName` command, i.e. `--no-window -prop persist.sys.language=en -prop persist.sys.country=US`.",
+            order = 5
     )
     var emulatorStartOptions: List<String>? = null
 
     @Parameter(
             names = arrayOf("--emulator-start-timeout-seconds"),
             required = false,
-            description = "Timeout to wait for emulator to finish boot. Default value is 180 seconds."
+            description = "Timeout to wait for emulator to finish boot. Default value is 180 seconds.",
+            order = 6
     )
     var emulatorStartTimeoutSeconds: Long? = null
 
     @Parameter(
             names = arrayOf("--redirect-logcat-to"),
             required = false,
-            description = "Path either relative or absolute to the file that will be used to redirect logcat of started emulator to. No redirection will happen if parameter is not presented.")
+            description = "Path either relative or absolute to the file that will be used to redirect logcat of started emulator to. No redirection will happen if parameter is not presented.",
+            order = 7
+    )
     var redirectLogcatTo: String? = null
 }
 
 fun parseArgs(rawArgs: List<String>): List<Args> {
-    if (PARAMETER_HELP_NAMES.firstOrNull { rawArgs.contains(it) } != null) {
+    if (COMMAND_HELP_NAMES.firstOrNull { rawArgs.contains(it) } != null) {
         JCommander(JCommanderArgs()).usage()
         System.exit(0)
     }
 
+    if (COMMAND_STOP_NAMES.firstOrNull { rawArgs.contains(it) } != null) {
+        stopAllEmulators().toBlocking().firstOrDefault(Unit)
```

Comment:
Oh, let's move it to main(), parseArgs() should definitely not be doing anything like that

Maybe return sealed class of parse result?